R/maraca.R

```diff
@@ -347,124 +426,116 @@ plot_maraca <- function(
     ) +
     ggplot2::guides(fill = "none")
 
+  # Add label to plot - maracaPlot
+  attr(plot, "label") <- "maracaPlot"
```

Comment:
ditto.

Reply:
Added maracaPlot to the class attributes